lib/lotus/generators/application/app/Gemfile.tt

```diff
@@ -10,6 +10,7 @@ gem 'lotus-validations', require: false, github: 'lotus/validations'
 gem 'lotus-helpers',     require: false, github: 'lotus/helpers'
 gem 'lotus-controller',  require: false, github: 'lotus/controller'
 gem 'lotus-view',        require: false, github: 'lotus/view'
+gem 'lotus-mailer',        require: false, github: 'lotus/mailer'
```

Comment:
please remove this line because lotus-mailer is an _optional_ dependency just like lotus-model